plugin/src/main/java/io/snyk/eclipse/plugin/utils/FileSystemUtil.java

```diff
@@ -19,18 +20,18 @@ public final class FileSystemUtil {
    * @return File
    */
   public static File getCliFile() {
-    return new File(getCliDirectory(), Platform.current().snykWrapperFileName);
+    return new File(new Preferences().getCliPath());
```

Comment:
This seem to fail unit tests, since preferences mock doesn't get passed here. Can we address this?